packages/core/lib/global-command-options.js

```diff
@@ -4,6 +4,11 @@ const options = {
     description:
       "Specify the network to use. Network name must exist in the configuration."
   },
+  from: {
+    option: "--from <account>",
+    description:
+      "Specify the account from which to make calls or send transactions."
```

Comment:
Not sure if it's worth adding this word here:

```suggestion
      "Specify the default account from which to make calls or send transactions."
```

Reply:
Adding the `default` word makes sense. Thanks for the suggestion, @gnidan.